src/plugins/console/server/lib/spec_definitions/js/aggregations.ts

```diff
@@ -320,13 +320,40 @@ const rules = {
     geohash_grid: {
       __template: {
         field: '',
-        precision: 3,
+        precision: 5,
       },
       field: '{field}',
       precision: { __one_of: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12] },
       size: 10,
       shard_size: 10,
     },
+    geohex_grid: {
```

Comment:
Should `bounds` also be supported ([docs](https://www.elastic.co/guide/en/elasticsearch/reference/master/search-aggregations-bucket-geohexgrid-aggregation.html#geohexgrid-options))?